### PR TITLE
Fix whitespace issues in the site content markdown

### PR DESCRIPTION
**Pull request by afbjorklund**
Courtesy of `markdownlint`, but nothing to show on:

`git show --ignore-all-space --ignore-blank-lines`

Remaining issues:

`find site/content/en/docs/ -name '*.md' | grep -v commands/ | xargs markdownlint`

```
      1 MD001/heading-increment/header-increment
      1 MD003/heading-style/header-style
      3 MD025/single-title/single-h1
      7 MD034/no-bare-urls
     26 MD040/fenced-code-language
      1 MD041/first-line-heading/first-line-h1
```

### CODE REVIEW

**Comment by medyagh** [comment]
@afbjorklund can we make sure our make target that generates this automatically doesn't overwrite it again with wrong format ?

**Comment by afbjorklund** [comment]
Probably a good idea, otherwise would have to run it again...

**Comment by afbjorklund** [comment]
The extra whitespace is hardcoded in "cobra" and "flags" :-(

File: site/content/en/docs/contrib/triage.md [hunk 1]
```diff
@@ -16,21 +16,23 @@ Triage is an important part of maintaining the health of the minikube repo.
...
 
 # Daily Triage
+
 Daily triage has two goals:
 
 1. Responsiveness for new issues
 1. Responsiveness when explicitly requested information was provided
 
-The list of outstanding items are at http://tinyurl.com/mk-tparty/daily-triage - it covers:
+The list of outstanding items are at http://tinyurl.com/mk-tparty/daily-triage- it covers:
```
Comment (priyawadhwa) [inline site/content/en/docs/contrib/triage.md:35]: I think the whitespace around this dash should still be there, maybe change to two dashes to that it's clear:

`trage -- it covers`

File: site/content/en/docs/handbook/controls.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ weight: 2
 description: >
   See minikube in action!
 aliases:
-  - /docs/examples/
+
```
Comment (priyawadhwa) [inline site/content/en/docs/handbook/controls.md:8]: I don't think we need a newline here

File: site/content/en/docs/handbook/controls.md [hunk 3]
```diff
@@ -5,7 +5,9 @@ weight: 2
...
+- /docs/examples/
+
```
Comment (priyawadhwa) [inline site/content/en/docs/handbook/controls.md:10]: Don't need this one either

File: site/content/en/docs/handbook/controls.md (hunk 2 above)
Comment (afbjorklund) [reply site/content/en/docs/handbook/controls.md:8]: site/content/en/docs/handbook/controls.md:9 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "- /docs/examples/"]

File: site/content/en/docs/handbook/controls.md (hunk 3 above)
Comment (afbjorklund) [reply site/content/en/docs/handbook/controls.md:10]: Just going off markdownlint

**Review by tstromberg — APPROVED** [review]

**Comment by tstromberg** [comment]
Thank you!

**Pull request merged**